src/Common/utils.ts

```diff
@@ -1,5 +1,38 @@
+import { Utils } from 'xpring-common-js/build/src'
```

Comment:
`import { Utils } from 'xpring-common-js`

No need to specify the exact path because this is exported from 'index.ts'

Reply:
Ah, sorry, I think the IDE did this on its own? I didn't type that out myself; will fix.